docs/source/private-data/private-data.md

```diff
@@ -151,6 +151,15 @@ documentation on [transaction flow](../txflow.html).
    their copy of the private state database and private writeset storage. The
    private data is then deleted from the `transient data store`.
 
+The [Fabric Gateway](../gateway.html) takes care of collecting the required set of endorsements
+on behalf of the client application.  It does this by running the first simulation on a peer in
+the gateway's organization and aggregating the relevant endorsement policies based on the read/write
+set generated by the chaincode. Client applications that submit transaction requests through
+the gateway using one of the [gateway SDKs](../gateway.html#software-development-kits-sdks)
+will benefit from this [endorser selection logic](../gateway.html#how-the-gateway-endorses-your-transaction-proposal).
+The client application can override this logic by explicitly [specifying which organizations](../gateway.html#targeting-specific-endorsement-peers)
```

Comment:
Suggest changing 160 to: "If any endorsement policy fails for a private data transaction, the transaction proposal fails without a retry. The client is then required to [explicitly specify each endorsing organization]."
And then delete 161.

Reply:
The application developer would need to make this decision when they are designing/coding their client application.  Your wording would suggest they would catch it and handle it as a runtime exception.  I'm not sure that's appropriate for this.